package.json

```diff
@@ -129,5 +129,10 @@
     "test:chrome": "npm run test -- --watch --browsers Chrome",
     "test:osx": "npm run test -- --browsers PhantomJS,Chrome,Firefox,Safari",
     "test:win": "npm run test -- --browsers PhantomJS,Chrome,Firefox,IE,IE9,IE10"
+  },
+  "dependencies": {
+    "@types/create-react-class": "^15.6.2",
```

Comment:
I think we shouldn't add this—usages of create-react-class should be replaced with class..extends rather than ported to TypeScript.

Reply:
I just realized that `create-react-class` isn't actually in our `package.json`. Is this something that React provides automatically? I needed to manually add it `yarn add --dev create-react-class` when upgrading the `react-modal` dependency. Strange.